graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -203,18 +202,44 @@ class SchemaGraph(object):
     on the graph. It also holds a fully denormalized schema for the graph.
     """
 
-    def __init__(self, schema_query_result):
+    def __init__(self, schema_data):
         """Create a new SchemaGraph from the given schema and index query results.
 
         Args:
-            schema_query_result: list of dicts describing the OrientDB schema, in OrientDB's format:
-                                 e.g. "SELECT * FROM (SELECT expand(classes) FROM metadata:schema)"
-                                 Required to already be topologically sorted with respect to the
-                                 inheritance hierarchy.
+            schema_data: list of dicts describing the classes in the OrientDB schema.
```

Comment:
Maybe mention as a hint that this is the way the data is structured in orientdb 2 and it's easy to get with a single query.